tests/unit/calendar/calendar_core.js

```diff
@@ -139,7 +139,7 @@ asyncTest( "keyboard handling", function() {
 			.focusGrid( element )
 			.simulate( "keydown", { keyCode: $.ui.keyCode.LEFT } );
 		setTimeout(function() {
-			$( ":focus" ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
+			$( document.activeElement ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
```

Comment:
Yeah, where it makes sense. In this case the focus should stay on the grid, when triggering tab events, it shouldn't. Currently the tests aren't explicit about that.